spec/workers/reactions/update_relevant_scores_worker_spec.rb

```diff
@@ -45,5 +45,10 @@
         worker.perform(Reaction.maximum(:id).to_i + 1)
       end.not_to raise_error
     end
+
+    it "uses a throttled call for syncing the reactions count" do
```

Comment:
@mstruve Added this spec now. I think since `ThrottledCall` has unit tests, it's enough to check that it receives the right arguments. I didn't want the spec to fail every time we update the interval, so I only verify the class of the second argument.

Reply:
Perfect! Thanks!